windowPreview.js

```diff
@@ -359,16 +359,16 @@ class WindowPreviewMenuItem extends PopupMenu.PopupBaseMenuItem {
                 ? Clutter.ActorAlign.START : Clutter.ActorAlign.END,
             y_align: Clutter.ActorAlign.START,
         });
-        this.closeButton.add_actor(new St.Icon({icon_name: 'window-close-symbolic'}));
+        this.closeButton.add_child(new St.Icon({icon_name: 'window-close-symbolic'}));
```

Comment:
Please change to `set_child` per 255ef10db9c330643188faca5ea82093f7ad0bb0

Reply:
Done. Thanks!